ports/samd/mcu/samd51/mpconfigmcu.mk

```diff
@@ -1,7 +1,16 @@
 MICROPY_VFS_LFS2 ?= 1
+MICROPY_VFS_FAT ?= 1
 
 SRC_S += shared/runtime/gchelper_m3.s
 
+SRC_C += \
+	fatfs_port.c \
+	extmod/modonewire.c \
+	drivers/dht/dht.c \
+
+SRC_QSTR += \
```

Comment:
This is no longer required (SRC_QSTR is now automatic).

Reply:
Done